.pylintrc

```diff
@@ -6,9 +6,14 @@ function-rgx=[a-z_][a-z0-9_]{2,50}$
 [MESSAGES CONTROL]
 
 disable=
+# Disabling "W" Warnings for stylistic problems or minor programming issues, as discussed on our stand up
+ W,
```

Comment:
I'd not disable all the warning messages. First, this PR is about errors. Second, there's many warning messages and when I look at them some of them are not good to dismiss, like:
`Assigning to function call which only returns None`
`Duplicate key %r in dictionary`
`Redefining built-in %r`
`Redefining name %r from outer scope (line %s)`
`Unreachable code`
`Unused import %s`
`Unused variable %r`